teleexception/__init__.py,sha256=Fz1y-Kx2hVMKRQR1U_DhbKpjdI1u24RK5DcKzNOqkbc,97
teleexception/status/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
teleexception/status/http_status.py,sha256=um25AKe04t9OHNT2MYikfrMM5nm3nTWlmrRpO7AzQME,12208
teleexception/status/status_exception.py,sha256=syVOIKg72y_Tjv_ndRuxIJ-ec_HGF3jDz_xqFj4WFQY,361
teleexception-0.1.5.dist-info/METADATA,sha256=Ne3tDBjbUSWn1S4RZ0zw0OSaHukO-Ry700qiD7uPCTs,809
teleexception-0.1.5.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
teleexception-0.1.5.dist-info/top_level.txt,sha256=8UE4OrNUAjQojJfld8TkzSD2RaYtO4XlXHzQIGBrTXU,14
teleexception-0.1.5.dist-info/RECORD,,
